project_task_stage_change_restriction/__init__.py

```
@@ -0,0 +1,4 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
```

Ruff (0.11.9)

`.models` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)